dask_sql/context.py

```diff
@@ -531,6 +531,29 @@ def register_model(
         schema_name = schema_name or self.schema_name
         self.schema[schema_name].models[model_name.lower()] = (model, training_columns)
 
+    def set_config(
+        self,
+        config_options: Union[Tuple[str, Any], Dict[str, Any]],
+        schema_name: str = None,
+    ):
+        """
+        Add configuration options to a schema.
+        A configuration option could be used to set the behavior of certain configurirable operations.
+
+        Eg: `dask.groupby.agg.split_out` can be used to split the output of a groupby agrregation to multiple partitions
```

Comment:
Should we add usage examples to these docstrings, or would users be interacting with this API through `ConfigContainer`?

Reply:
Thanks, I need to update the doctoring here